Gruntfile.js

```diff
@@ -689,23 +713,31 @@ module.exports = function(grunt) {
 	}
 
 	/**
-	 * Transform JS inside some file into a `case` inside a `switch` statement
+	 * Transform JS inside some file into an object,
+	 * keyed by locale and valued by a function what loads formatting that for that locale,
 	 * whilst at the same time compiling the list of supported language codes.
 	 * @param {Set<string>} codes
 	 * @returns {Function}
 	 */
 	function getlocalesprocessor(codes) {
 		return (src, file) => {
 			const code = file.match(/ts-lang-(.*)\.js/)[1];
+			let locale;
+
+			try {
+				locale = Locale.parseStrict(code);
+			} catch (e) {
+				throw new Error(`Invalid locale could not be parsed (${code}): ${e}`);
```

Comment:
Maybe it's better to use https://gruntjs.com/api/grunt.fail

Reply:
Done :)